ap/plotting/plotter.py

```diff
@@ -14,45 +14,46 @@
 mplhep = maybe_import("mplhep")
 
 
-def draw_error_stairs(ax: plt.Axes, h: hist.Hist, kwargs: Optional[dict] = None) -> None:
-    if kwargs is None:
-        kwargs = {}
-    norm = kwargs.pop("norm", 1)
-    values = h.values() / norm
-    error = np.sqrt(h.variances()) / norm
+def draw_error_stairs(ax: plt.Axes, h: hist.Hist, norm: float = 1.0, **kwargs) -> None:
```

Comment:
might be a good point to choose a better name since we are not using `ax.stairs` anymore. How about `draw_error_bands`?

Reply:
Good point! Changed.